pinot-segment-local/src/test/java/org/apache/pinot/segment/local/utils/nativefst/ImmutableFSTTest.java

```diff
@@ -72,6 +74,7 @@ private static void verifyContent(FST fst, List<String> expected) {
     }
     actual.sort(null);
     assertEquals(actual, expected);
+    close(fst);
```

Comment:
is this correct? this `close` method won't be called in case the `assertEquals` above fails.